include/rdma/fi_log.h

```diff
@@ -35,6 +35,8 @@
 #ifndef _FI_LOG_H_
 #define _FI_LOG_H_
 
+#include "config.h"
```

Comment:
> Where is this file coming from?

This file is generated by `configure` (technically by `config.status`).  It's the primary way that the results of configure tests are communicated to the C code that might conditionally compile based upon those tests.

> Or why isn't is something like this (from other source files):
```c
#if HAVE_CONFIG_H
#  include <config.h>
#endif /* HAVE_CONFIG_H */
```

Because:
1. That stanza is actually incorrect, it should be `#ifdef` instead of `#if`
2. We always invoke `AC_CONFIG_HEADERS` in `configure.ac`, which means that [we will always be compiled with `-DHAVE_CONFIG_H`](http://www.gnu.org/software/autoconf/manual/autoconf.html#Configuration-Headers) and there's not any case where we should ever be missing `config.h`.